lib/serializer.ts

```diff
@@ -159,6 +159,9 @@ export class Serializer {
         // so let's do the more user-friendly thing and return an empty list.
         responseBody = [];
       }
+      if (mapper.defaultValue) {
```

Comment:
are there any subtle bugs this could bring in related to falsy default values?

Reply:
Thanks for pointing this out. Have changed this to specifically check for `undefined`.